classes/Domain/Services/ResetEmailer.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace OpenCFP\Domain\Services;
+
+class ResetEmailer
```

Comment:
From the API-perspective, I think moving towards (and we're 120% outside the scope of this work now) that type of model offers some pretty interesting integration opportunities for connected systems. Imagine System A being able to ask an OpenCFP instance, "Hey! What are all the things that `yitznewton` did since yesterday?" ... "Hey! I know I know... I was down for a few weeks, could you be an awesome OpenCFP API and let me know all the account registrations that happened since 3 weeks ago?" etc.